kytos/of.l2lsloop/main.py

```diff
@@ -124,7 +124,8 @@ def handle_message_in_packet_event(self, event):
             event_out = events.KycoMessageOutFlowMod(event.dpid, content)
             self.controller.buffers.msg_out.put(event_out)
         elif (flood_hash not in switch.flood_table or
-                (now() - switch.flood_table[flood_hash]).microseconds > FLOOD_TIMEOUT):
+                (now() - switch.flood_table[flood_hash]).microseconds >
+                POOLING_TIME):
```

Comment:
Yes, @cemsbr I agree. But anyway here is FLOOD_TIMEOUT as @diraol said. I cannot accept with POLLING_TIME this will break the logic. POLLING_TIME is too smaller than FLOOD_TIMEOUT.